packages/documentation-site/patternfly-docs/content/accessibility/code-editor/code-editor.md

```diff
@@ -3,42 +3,68 @@ id: Code editor
-monaco code editor. Each control should have an aria-label since they are designed to display only an icon with a 
-Tooltip. The Tooltips are not announced by the screen reader, so the aria-label must be descriptive and clear.
-
-The following props/attributes have been added for you or are customizable in PatternFly:
-
-| React prop | React component it should be applied to | Which HTML element it appears on in markup | Reason used |
-| -- | -- | -- | -- |
-| copyButtonAriaLabel | CodeEditor | `.pf-v5-c-code-editor__controls button` | Aria-label for the copy button |
-| downloadButtonAriaLabel | CodeEditor | `.pf-v5-c-code-editor__controls button` | Aria-label for the download button |
-| uploadButtonAriaLabel | CodeEditor | `.pf-v5-c-code-editor__controls button` | Aria-label for the upload button |
-| 'aria-label' | CodeEditorControl | `.pf-v5-c-code-editor__controls button` | Aria-label for a custom control button |
-| shortcutsPopoverButtonText | CodeEditor | `.pf-v5-c-code-editor__keyboard-shortcuts` | Text to show in the button to open the shortcut popover |
-| shortcutsPopoverProps | CodeEditor | `.pf-v5-c-code-editor__controls button` | Properties for the shortcut popover. Should contain 'bodyContent' to provide keybaord shortcuts built into the code editor |
-
-The monaco code editor comes with some keyboard shortcuts built in, so it is recommended that consumers communicate 
-information about the following built in key commands:
-
-| Opt + F1 | Accessibility helps |
-| F1 | View all editor shortcuts |
-| Ctrl + Space | Activate auto complete |
-| Cmd + S | Save |
-
-Additionally, when implementing a **code editor**, a developer can add additional keyboard controls by adding a command
-to the `editor` parameter passed via the `onEditorDidMount` prop. For example:
-```
-onEditorDidMount = (editor, monaco) => {
-    editor.addCommand(monaco.KeyMod.Shift | monaco.KeyCode.Space, () => {console.log("hello")});
-};
-```
\ No newline at end of file
+
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+import { Link } from '@patternfly/documentation-framework/components/link/link';
+
+## Accessibility
+
+To implement an accessible PatternFly **code editor**:
+
+- Ensure any code editor controls can be navigated to and interacted with via keyboard and other assistive technologies such as screen readers.
+- Provide descriptive `aria-label` and tooltips to code editor controls that only display an icon. Refer to our [tooltip accessibility](/components/tooltip/accessibility) documentation for more information on accessible tooltips.
+- Update a tooltip's content if the code editor control triggers an action that isn't immediately obvious to users, such as clicking a "copy to clipboard" control.
+- Provide a shortcuts menu that can be accessed via mouse, keyboard, or other assistive technologies such as screen readers, especially when using custom, unfamiliar shortcuts.
+- Include keyboard shortcuts for both PC and Mac keyboards
```

Comment:
This isn't something we currently enforce, but I noticed in the shortcuts example that we list keys for Mac, but not PC. Would require some updating in React, but would be beneficial to try updating since the opposite OS keys may not be immediately known .